src/platforms/gbm-kms/server/kms/display_buffer.cpp

```diff
@@ -101,6 +101,12 @@ geom::Rectangle mgg::DisplayBuffer::view_area() const
     return area;
 }
 
+auto mgg::DisplayBuffer::pixel_size() const -> geom::Size
+{
+    // All the outputs (are meant to) have the same size; KMSOutput::size() gives the size in pixels
```

Comment:
[non-blocking] This is a consequence of only sharing a display buffer between compatible outputs. Sounds like it can be asserted somewhere?